tests/test_messages.py

```diff
@@ -45,9 +45,7 @@ def test_balance_proof_address_setter(get_random_bp):
 def test_balance_proof():
     # test balance proof with computed balance hash
     balance_proof = BalanceProof(
-        channel_identifier=ChannelIdentifier(
-            '0x3131313131313131313131313131313131313131313131313131313131313131',
-        ),
+        channel_identifier=0x3131313131313131313131313131313131313131313131313131313131313131,
```

Comment:
Should we replace those with something more realistic? And decimal?

Reply:
replaced it with a RNG in uint256 range